typings/index.d.ts

```diff
@@ -3014,6 +3036,8 @@ declare module 'discord.js' {
 
   type PermissionResolvable = BitFieldResolvable<PermissionString, bigint>;
 
+  type PermissionOverwriteResolvable = Snowflake | PermissionOverwrites;
```

Comment:
why not also add `GuildMember`, `User`, and `Role` types here too (could also use the respective `Resolvable` types too i suppose

Reply:
out of the scope of this pr, perhaps a separate one (?)

Reply:
this type is added in this PR, it would make sense to resolve a `PermissionOverwrite` from the types above, so why not add that functionality right off the bat ?

Reply:
oh nvm, i misread what you said, disregard. will do, good catch, thanks!